src/utils/api.ts

```diff
@@ -53,38 +55,59 @@ function redirectOccurred(value: AxiosResponse<TeslerResponse>) {
  * @param callContext 
  */
 function onErrorHook(error: AxiosError, callContext?: ApiCallContext) {
-    if (error.response?.status === 418) {
-        const typedError = error.response.data as OperationError
-        if (typedError.error.popup) {
-            const businessError: BusinessError = {
-                type: ApplicationErrorType.BusinessError,
-                message: typedError.error.popup[0]
+    if (error.response) {
+        switch (error.response.status) {
+            case 401 : {
+                getStoreInstance().dispatch($do.logoutDone(null))
+                historyObj.push('/')
+                break
             }
-            getStoreInstance().dispatch($do.showViewError({ error: businessError }))
-            if (typedError.error.postActions?.[0]) {
-                const widget = getStoreInstance().getState().view.widgets.find(item => item.name === callContext.widgetName)
-                const bcName = widget?.type
-                getStoreInstance().dispatch($do.processPostInvoke({
-                    bcName,
-                    postInvoke: typedError.error.postActions[0],
-                    widgetName: widget.name
-                }))
+            case 418 : {
+                const typedError = error.response.data as OperationError
+                if (typedError.error.popup) {
+                    const businessError: BusinessError = {
+                        type: ApplicationErrorType.BusinessError,
+                        message: typedError.error.popup[0]
+                    }
+                    getStoreInstance().dispatch($do.showViewError({error: businessError}))
+                    if (typedError.error.postActions?.[0]) {
+                        const widget = getStoreInstance().getState().view.widgets.find(item => item.name === callContext.widgetName)
+                        const bcName = widget?.type
+                        getStoreInstance().dispatch($do.processPostInvoke({
+                            bcName,
+                            postInvoke: typedError.error.postActions[0],
+                            widgetName: widget.name
+                        }))
+                    }
+                }
+                break
+            }
+            case 409 : {
+                const notificationMessage = error.response.data.error?.popup?.[0] || ''
+                openButtonWarningNotification(
+                    notificationMessage,
+                    'OK',
+                    0,
+                    () => {Observable.empty()},
```

Comment:
Observables dObservable.empty is returned in our observables chains to terminate epic reaction to observed action.  
In this case nothing is returned and nothing is observed so it should be just `() => {}`

Reply:
Fixed